packages/terra-clinical-site/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 
 ### Changed
 * Update Header example
+* Added Subheader example
```

Comment:
@emilyrohrbough just to be sure before I push my changes:

should this file say something like this:

```
### Added
* Example to render subheader component
```
as we are only adding example file in terra-clinical-site.
and CHANGELOG.md for terra-clinical-header should say something like below:

```
### Added
* isSubheader boolean prop to allow rendering of subheader component
```